packages/plugin-vue-jsx/README.md

```diff
@@ -17,7 +17,23 @@ export default {
 
 ## Options
 
-See [@vue/babel-plugin-jsx](https://github.com/vuejs/jsx-next).
+### include
+
+Type: `<string | RegExp> | string | RegExp | null;`
+
+Default: `/\.[jt]sx$/`
+
+if you want to add extensions option to vue-jsx plugin, you can use `include`
```

Comment:
```suggestion
A [picomatch pattern](https://github.com/micromatch/picomatch), or array of patterns, which specifies the files the plugin should operate on.
```

Sorry I forgot to submit this review earlier. Was proposing to use this kind of description as the rollup plugins use it too.